tests/python/gaia-ui-tests/gaiatest/tests/graphics/test_play_3gp_video.py

```diff
@@ -0,0 +1,67 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+
+from gaiatest.gaia_graphics_test import GaiaImageCompareTestCase
+from gaiatest.apps.videoplayer.app import VideoPlayer
+
+
+class TestPlay3GPVideo(GaiaImageCompareTestCase):
+    def setUp(self):
+        GaiaImageCompareTestCase.setUp(self)
+
+        # add video to storage
+        self.push_resource('VID_IC_0001.3gp')
+
+    def test_play_3gp_video(self):
+        """https://moztrap.mozilla.org/manage/case/2478/"""
+
+        self.take_screenshot()
+
+        video_player = VideoPlayer(self.marionette)
+        video_player.launch()
+        video_player.wait_for_thumbnails_to_load(1, 'Video files found on device: %s' % self.data_layer.video_files)
+
+        # Assert that there is at least one video available
+        self.assertGreater(video_player.total_video_count, 0)
+        first_video_name = video_player.first_video_name
+        self.take_screenshot()
+
+        # See bug 1109203, current_audio_channel switches context to the system
+        # app, so we need to return it back to the displayed app
+        self.assertEqual('none', self.data_layer.current_audio_channel)
+        self.apps.switch_to_displayed_app()
```

Comment:
aha, thought I needed  self.apps.switch_to_displayed_app() for the script to work, but guess that is only for the audio. removed.